orca-core/src/test/groovy/com/netflix/spinnaker/orca/StageResolverSpec.groovy

```diff
@@ -67,4 +72,16 @@ class StageResolverSpec extends Specification {
   @StageDefinitionBuilder.Aliases("notAliased")
   class AliasedStageDefinitionBuilder implements StageDefinitionBuilder {
   }
+
+  class SimpleStage implements SimpleStage {
```

Comment:
Let's uh, rename this? `TestSimpleStage` or something. Having a class the same name as an interface is mega confusing.